### PR TITLE
CODE RUB: Bugs

### PR DESCRIPTION
**Pull request by davidhayes03**

### CODE REVIEW

File: ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/components/accessAudits/accessAuditTable.tsx [hunk 1]
```diff
@@ -47,65 +47,73 @@ const AccessAuditTable: FunctionComponent<AccessAuditTableProps> = () => {
...
+                ) : (
+
+                    <InfiniteScroll loading={isLoading || showSpinner} hasNextPage={hasNextPage || false} loadMore={fetchNextPage}>
 
-                            <Table striped bordered hover variant="light" responsive>
-                                <thead>
+                        <Table striped bordered hover variant="light" responsive>
+                            <thead>
+                                <tr>
+                                    {/* <th>Identifier</th>*/}
```
Comment (cjdutoit) [inline ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/components/accessAudits/accessAuditTable.tsx:75]: Should this still be here?

File: ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/components/accessAudits/accessAuditRowView.tsx [hunk 2]
```diff
@@ -13,12 +13,11 @@ const AccessAuditRowView: FunctionComponent<AccessAuditRowProps> = (props) => {
     return (
         <>
             <tr>
-                <td><small>{accessAudit.pseudoIdentifier}</small></td>
+                {/*<td><small>{accessAudit.pseudoIdentifier}</small></td>*/}
```
Comment (cjdutoit) [inline ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/components/accessAudits/accessAuditRowView.tsx:16]: Should this still be here?

File: ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/components/accessAudits/accessAuditTable.tsx (hunk 1 above)
Comment (davidhayes03) [reply ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/components/accessAudits/accessAuditTable.tsx:75]: yes will uncomment when we have merged your change in

File: ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/components/accessAudits/accessAuditRowView.tsx (hunk 2 above)
Comment (davidhayes03) [reply ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Client/src/components/accessAudits/accessAuditRowView.tsx:16]: yes will uncomment when we have merged your change in

**Review by cjdutoit — APPROVED** [review]

**Pull request merged**